src/openforms/appointments/tests/test_pdf.py

```diff
@@ -31,8 +32,11 @@ def test_appointment_info_is_included(self):
         self.assertIn("Afspraakinformatie", html)
         self.assertIn("Test product 1", html)
         self.assertIn("Test location", html)
+        self.assertIn("Test address", html)
         self.assertIn("Datum en tijd", html)
 
+        self.assertNotIn(escape("<h1>"), html)
```

Comment:
yeah so we want to see the escaped data here, plugin developers need to explicitly `mark_safe` or `format_html` that stuff IMO.

That also requires updating the comment for `openforms.appointments.base.AppointmentDetails.other`:

> \# These are typically key/values-pairs where both the key and value are
> \# considered *not* to be HTML-safe. Plugins need to explicitly mark trusted
> \# content as safe.